src/main/java/lotto/domain/LottoMachine.java

```diff
@@ -0,0 +1,35 @@
+package lotto.domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class LottoMachine {
+
+    private static final int LOTTO_NUMBER_START_INDEX = 0;
+    private static final int LOTTO_NUMBER_END_INDEX = 6;
+    private static final int LOTTO_NUMBER_LOWER_BOUND = 1;
+    private static final int LOTTO_NUMBER_UPPER_BOUND = 46;
+
+    private final List<LottoNumber> numbers;
+
+    public LottoMachine() {
+        numbers = new ArrayList<>();
+        for (int i = LOTTO_NUMBER_LOWER_BOUND; i < LOTTO_NUMBER_UPPER_BOUND; i++) {
+            numbers.add(new LottoNumber(i));
+        }
+    }
+
+    public List<Lotto> makeLottoTickets(int count) {
+        ArrayList<Lotto> lottoTickets = new ArrayList<>();
```

Comment:
```suggestion
       List<Lotto> lottoTickets = new ArrayList<>();
```
이런식으로 하면 의존성을 낮출수 있겠네요 👍

Reply:
당연히 List로 선언했을줄 알았늗데 ArrayList로 선언했네요ㅎㅎ 다음부터는 주의하겠습니다 🥲